libs/testing/e2e/src/lib/config/playwright-config.ts

```diff
@@ -0,0 +1,54 @@
+import { defineConfig, devices } from '@playwright/test'
+
+interface GlobalConfigParams {
+  webServerUrl: string
+  port?: number
+  command: string
+  cwd?: string
+  timeoutMs?: number
+}
+
+export const createGlobalConfig = ({
+  webServerUrl,
+  port,
+  command,
+  cwd = '../../../',
+  timeoutMs = 5 * 60 * 1000,
+}: GlobalConfigParams) => {
+  return defineConfig({
+    testDir: 'e2e',
+    fullyParallel: true,
+    forbidOnly: !!process.env.CI,
+    retries: process.env.CI ? 2 : 0,
+    workers: process.env.CI ? 1 : undefined,
+    reporter: 'html',
```

Comment:
Should be json or something (for DataDog) when running in CI

Reply:
Addressing in https://github.com/island-is/island.is/pull/16816